lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -115,18 +120,33 @@ class SemanticsNodeUpdate {
   /// See [ui.SemanticsUpdateBuilder.updateNode].
   final String label;
 
+  /// See [ui.SemanticsUpdateBuilder.updateNode].
```

Comment:
(here and elsewhere): The API doc on that methods doesn't seem to mention this field.

Reply:
Also: Let's document that the content of these lists must not change.